src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -187,7 +166,7 @@
 @private lroSampleCode(apiMethod, sample)
   operation = {@methodCallSampleCode(apiMethod, sample.sampleInitCode)}
 
-  print('Waiting for operation to complete...')
+  print(u'Waiting for operation to complete...')
```

Comment:
Python3 has Unicode strings by default, it seems, so the `u` prefix is unnecessary.

https://stackoverflow.com/a/2464968
https://docs.python.org/3/howto/unicode.html

Reply:
@beccasaurus It seems Python 3 does not like the `'u'`?